src/components/AdvertBanner/index.tsx

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from 'react'
+import { AdvertWrapper } from 'src/components/AdvertBanner/styles'
+
+export const AdvertBanner = ({
+  children,
+}: {
+  children?: ReactNode
```

Comment:
we have type for children in global types file and types should be in separate files